src/HtmlReader/index.tsx

```diff
@@ -141,15 +162,33 @@ export default function useHtmlReader(args: ReaderArguments): ReaderReturn {
       },
       userSettings: userSettings,
       api: {
-        getContent: getContent as any, //TODO: fix this casting,
+        getContent: getContent as GetContent,
+        updateCurrentLocation: async (location: Locator) => {
+          // This is needed so that setBookBoundary has the updated "reader" value.
+          dispatch({ type: 'LOCATION_CHANGED', location: location });
+          return await location;
+        },
         onError: function (e: Error) {
           setError(e);
         },
-      } as any, //TODO: fix this casting,,
+      } as NavigatorAPI,
     }).then((reader) => {
       dispatch({ type: 'SET_READER', reader });
+      enableResizeEvent(reader, dispatch);
     });
-  }, [webpubManifestUrl, getContent, injectables, injectablesFixed]);
+  }, [
+    webpubManifestUrl,
+    getContent,
+    injectables,
+    injectablesFixed,
+    state.isScrolling,
```

Comment:
I don't think we want to re-initialize the reader every time the `state.isScrolling` state changes. It should already be updated when it changes. If you put it in here so you could use the default value I think you should bring only the default in to this effect, not the state variable

Reply:
Updated, thank you for catching this!